### PR TITLE
a few points

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/Developer/Roundtrip.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ originally created files.
 This is, where "Roundtrip mode" is needed. It aims to preserve your manual changes and applying the new model
 configuration at the same time.
 
-The roundtrip mode is enabled by default. To disable it see the ExtensionBuilder Configuration :doc:`/Configuration/Index`
+The roundtrip mode is enabled by default. To disable it see the Extension Builder Configuration :doc:`/Configuration/Index`
```
Comment (simonschaufi) [inline Documentation/Developer/Roundtrip.rst:19]: I'm not sure about this as ExtensionBuilder is the product name

Comment (franzholz) [reply Documentation/Developer/Roundtrip.rst:19]: ExtensionBuilder has been written as "Extension Builder" before. So the same name should be used everywhere.

**Comment by nicodh** [comment]
Is there a reason why all these PRs should go into 8.7 branch instead of master?

**Comment by franzholz** [comment]
The 8.7 branch is published. It should go in both branches.

**Pull request merged**